frontend/near-api.js

```diff
@@ -0,0 +1,73 @@
+import { connect, keyStores, WalletConnection } from 'near-api-js'
+import { parseNearAmount } from 'near-api-js/lib/utils/format'
+import { getConfig } from './config'
+
+const nearConfig = getConfig(process.env.NODE_ENV || 'development')
+
+// Initialize contract and set global variables
+export async function initContract() {
+  // Initialize connection to the NEAR testnet
```

Comment:
```suggestion
  // Initialize connection to the NEAR blockchain
```
based on getConfig it can be several networks